### PR TITLE
Fix LGTM Warnings

### PR DESCRIPTION
**Pull request by ryanmccombe**
Resolves 4 issues (1359 - 1362) in the infra repo

### CODE REVIEW

File: src/app/containers/LinkedData/getAboutTagsContent.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const checkType = types => {
 };
 
 const checkSameAs = uris => {
-  const sameAs = uris.filter(uri => uri.includes('http://dbpedia.org'));
+  const sameAs = uris.filter(uri => uri.startsWith('http://dbpedia.org'));
```
Comment (FK78) [inline src/app/containers/LinkedData/getAboutTagsContent.js]: Nice work, just one comment.

Github seems to have integrated LGTM and is still complaining about "Incomplete URL substring sanitization".

Comment (andrewscfc) [reply src/app/containers/LinkedData/getAboutTagsContent.js]: Is this error warning that someone could deploy a service to: http//dbpedia.org.nastypersonurl.com and then they could pass validation using logic like this? As it happens this logic isn't used for this sort of validation so there's no risk here to my knowledge.

Does it just need a slash adding on the end: `'http://dbpedia.org/'`?

**Review by andrewscfc — COMMENTED** [review]
All looks good just one outstanding warning to fix

File: src/app/containers/LinkedData/getAboutTagsContent.js (hunk 1 above)
Comment (FK78) [reply src/app/containers/LinkedData/getAboutTagsContent.js]: Adding a trailing slash may fix the error.

**Review by jroebu14 — APPROVED** [review]

**Review by rebeccamcginn — APPROVED** [review]

**Pull request merged**